### PR TITLE
snippets: Add YAML

### PR DESCRIPTION
**Pull request by adrienverge**
### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

### CODE REVIEW

File: data/snippets/yaml.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+```
+$ echo "---
+dict :
+  key:
+        value
+" >> test.yaml
+$ coala --files test.yaml --bears YAMLLintBear
```
Comment (yukiisbored) [inline data/snippets/yaml.md]: I think you should add `-I` or `--ignore-config` here

Comment (adrienverge) [inline data/snippets/yaml.md]: Sorry, but why? [Other examples](https://github.com/coala/landing-frontend/tree/a99aac1/data/snippets) don't use this option.

Comment (sils) [inline data/snippets/yaml.md]: @yukiisbored we don't add this by design so people do see the warning about the `--save` flag, it's a very conscious part to make the learning curve of coala easier for new users

**Comment by sils** [comment]
ack https://github.com/coala/landing-frontend/pull/134/commits/cccee55c7da25a363333c322e724d67b31e9a112

Thanks a lot @adrienverge ! It's a good example so let's get it in. @yukiisbored if you disagree with this decision to not use `-I` please file an issue because we'd have to change it everywhere, this PR would be the wrong place.

**Comment by sils** [comment]
ack cccee55

**Comment by sils** [comment]
@rultor merge

**Comment by yukiisbored** [comment]
Please rebase

**Comment by adrienverge** [comment]
Rebased!

**Comment by meetmangukiya** [comment]
ack 988fc2a

**Comment by meetmangukiya** [comment]
@rultor merge

**Pull request merged**